components/telemetry-operator/api/v1alpha1/logpipeline_types.go

```diff
@@ -137,6 +138,7 @@ func filterOutCondition(conds []LogPipelineCondition, condType LogPipelineCondit
 //+kubebuilder:resource:scope=Cluster
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[-1].type`
+//+kubebuilder:printcolumn:name="Unsupported-Plugins",type=boolean,JSONPath=`.spec.enableUnsupportedPlugins`
```

Comment:
Let's name the column "Unsupported-Plugin" since it contains only one value per row